packages/dx-react-chart/demos/overview/multiple-axes.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Overview
+
+The DevExtreme React Chart is a component that visualizes data using a predefined (Bar, Line, Area, Scatter, Pie, and more) or custom series type. You can visualize several series of the same type separately or together in a stack. If you want to visualize different data on the same chart, you can create a chart with multiple axes. The Chart enables you to customize the appearance of all its elements including axes, a legend, a grid, and a title. Explore our demos to define how to configure the chart for your project.
```

Comment:
"Explore our demos to define how to configure the chart for your project." Explore to define?